tests/test_compare_eclipse.cpp

```diff
@@ -0,0 +1,173 @@
+#include "test_compare_eclipse.hpp"
+
+int main(int argc, char** argv) {
+    char* gridFile1 = argv[1];
+    char* gridFile2 = argv[2];
+    char* unrstFile1 = argv[3];
+    char* unrstFile2 = argv[4];
+
+    if (argc != 5) {
+        printHelp();
+    }
+    else {
```

Comment:
this should be surrounded by a try { with a catch() at the end. see e.g. https://github.com/OPM/opm-simulators/blob/master/opm/autodiff/FlowMain.hpp#L132